dist.ts

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env -S node --loader @swc-node/register/esm
+import fse from "fs-extra";
+const { mkdtemp, copySync, mkdirpSync } = fse;
+import { rimraf } from "rimraf";
+import { join, dirname } from "path";
+import { tmpdir } from "os";
+import { fileURLToPath } from "url";
+import tar from "tar";
+
+console.log(`Packaging for distribution...`);
+
+const version = process.env.npm_package_version;
+if (version == null) {
+  throw new Error("Package version undefined.");
+}
+
+const srcDir = dirname(fileURLToPath(import.meta.url));
+
+// Set up a temporary directory
+mkdtemp(join(tmpdir(), "fluidqueue-"), async (err, directory) => {
```

Comment:
Is there a reason to use `fluidqueue-` instead of `fluid-queue-` (like everywhere else)?
(This doesn't really matter)